authentication/account/service/user.go

```diff
@@ -181,9 +181,10 @@ func (s *userServiceImpl) NotifyIdentitiesBeforeDeactivation(ctx context.Context
 
 // ListIdentitiesToDeactivate lists the identities to deactivate
 func (s *userServiceImpl) ListIdentitiesToDeactivate(ctx context.Context) ([]repository.Identity, error) {
-	since := time.Now().Add(-s.config.GetUserDeactivationInactivityPeriod()) // remove 'n' days from now (default: 31)
+	since := time.Now().Add(-s.config.GetUserDeactivationInactivityPeriod())                                                                    // remove 'n' days from now (default: 31)
```

Comment:
Would it be possible to include the unit type in the configuration parameter name, in order to reduce confusion about what kind of value to configure here?  For example, `GetUserDeactivationInactivityPeriodDays`

Reply:
sure, that's a good idea, let me do that.

Reply:
done in a8be694